src/components/Filter/components/FilterBase/FilterBase.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { forwardRef, ReactNode } from 'react';
+
+import useTheme from '../../../../hooks/useTheme';
+import { useTypeColorToColorMatch } from '../../../../hooks/useTypeColorToColorMatch';
+import { generateTestDataId } from '../../../../utils/helpers';
+import Icon from '../../../Icon';
+import { AcceptedIconNames } from '../../../Icon/types';
+import {
+  buttonSpanStyle,
+  buttonStyle,
+  buttonWrapperStyle,
+  childrenWrapperStyle,
+  dividedButtonStyle,
+  divider,
+  labelSpanStyle,
+  valueSpanStyle,
+  wrapperStyle,
+} from '../../Filter.style';
+import { FilterType, StyleType } from '../../types';
+
+// eslint-disable-next-line react/display-name
+export const FilterBase = forwardRef<
+  HTMLButtonElement,
+  {
+    dataTestId?: string;
```

Comment:
i would expect these to be a type like we do it

Reply:
i would also expect these to be the same for filter. Because there you use these props as well